Add /decodetracks endpoint support

james7132 opened the pull request:
In addition to the `Client/Node.build_track` method, this adds the corresponding POST to `/decodetracks` to batch decode track identifiers. Useful if there are a large number of tracks that need to be decoded.

The definition of the endpoint can be found here: https://github.com/Frederikam/Lavalink/blob/18ee6778674b7f9a817893b676a91a9b96e642f3/LavalinkServer/src/main/java/lavalink/server/player/AudioLoaderRestHandler.java#L140

EvieePy commented:
I'm still on the fence about this, if this was to be implemented I would consider removing the current `build_track` and replacing it with this.

In either case it might also be better suited to an argument list of tracks rather than a literal list.

james7132 commented:
Removing the original could be a breaking change, unless the method name remains the same.

I agree that an argument list would be a better interface than an actual list.

EvieePy commented:
Yes it would be a breaking change which is why I'm on the fence. Having `build_tracks` take in an argument list means that when passing only one track the behaviour would be identical to the current build track anyways, if you know what I mean.

james7132 commented:
Deleted the old version with a singular argument. Going to keep the name as `build_tracks` since the return type has changed too, which also is a breaking change. Cleaned up the doc comments. Please take another look.

james7132 commented:
Poke to review this again.

EvieePy commented:
Closing this as  [Branch 1.0](https://github.com/PythonistaGuild/Wavelink/tree/1.0) now supersedes master. Feel free to open a new PR should you feel this is a missing feature and direct it to 1.0
